README.md

```diff
@@ -32,22 +44,26 @@ It is defined using a [docker-compose file](../docker-compose.yml) for developme
 
 ### The storage & transcoding environment
 
-Source files (video, subtitles,...) are directly uploaded to an `S3` bucket by instructors, which triggers `MediaConvert` to generate all necessary video files (various formats and fragments & manifests for adaptive-bitrate streaming) into a destination `S3` bucket. Those files are then served through the `CloudFront` CDN.
+Source files (video, documents, subtitles,...) are directly uploaded to an `S3` bucket by instructors. Depending the uploaded resource a lambda will be triggered to do diffenrent jobs:
```

Comment:
```suggestion
Source files (video, documents, subtitles,...) are directly uploaded to an AWS `S3` bucket by instructors. Depending on the uploaded resource, a lambda pipeline will be triggered as follow:
```

Reply:
No this is not a pipeline but a list of what can be triggered

Reply:
Ok then, what about: "a lambda collection can be triggered"

Reply:
There is only one lambda (`lambda-encode`) responsible to do different tasks depending the uploaded resource.

Reply:
Ok, then it's not explicit in your original documentation: it should state that one lambda triggers other lambdas (if I understand well this time).

Reply:
No you don't. Only one is triggered (`lambda-encode`) and depending the resource you uploaded then a specific part of this lambda (still `lambda-encode`) is executed. This lambda never trigger other lambda. The other lambda are triggered by other mechanisms and we don't treat them in the readme.

Reply:
Looks like this README is not that straightforward... maybe a schema would help here.